azure-storage-blob/src/Blob/BlobSharedAccessSignatureHelper.php

```diff
@@ -90,7 +90,7 @@ public function generateBlobServiceSharedAccessSignatureToken(
         $signedPermissions,
         $signedExpiry,
         $signedStart = "",
-        $signedIP = "",
+        $signedIP = null,
```

Comment:
"" is the default parameter value for `$signedIP`, and `$buildOptQueryStr ` will not append it to the query string if it's "". Why we need change it to `null`?

```
$buildOptQueryStr = function ($string, $abrv) {
            return $string === '' ? '' : $abrv . $string;
        };
```